package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "butterfly",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
@lacosta2 it looks like this extra `package-lock.json` file was added in the root directory. Can you delete it before merging this pull request?

By the way, do you know how it got there? Did the GitPod startup steps go wrong? Or did you run `npm install` in the root directory?

CC @phodonou